modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -130,6 +130,53 @@ export class {{classname}} {
 {{/operation}}
 {{/useHttpClient}}
 
+    private addToHttpParams(
+        httpParams: {{#useHttpClient}}HttpParams{{/useHttpClient}}{{^useHttpClient}}URLSearchParams{{/useHttpClient}},
+        value: any, key?: string): {{#useHttpClient}}HttpParams{{/useHttpClient}}{{^useHttpClient}}URLSearchParams{{/useHttpClient}}
+    {
+        if (typeof value === "object") {
+            {{#useHttpClient}}httpParams = {{/useHttpClient}}this.addToHttpParams1(httpParams, value);
+        } else {
+            {{#useHttpClient}}httpParams = {{/useHttpClient}}this.addToHttpParams1(httpParams, value, key);
+        }
+        return httpParams;
+    }
+
+    private addToHttpParams1(
+        httpParams: {{#useHttpClient}}HttpParams{{/useHttpClient}}{{^useHttpClient}}URLSearchParams{{/useHttpClient}},
+        value: any, key?: string): {{#useHttpClient}}HttpParams{{/useHttpClient}}{{^useHttpClient}}URLSearchParams{{/useHttpClient}}
+    {
+        if (typeof value === "object") {
+            if (this.isArrayLike(value)) {
+                (value as []).forEach( elem => httpParams = this.addToHttpParams1(httpParams, elem, key));
+            } else if (value instanceof Date) {
+                if (key != null) {
+                    {{#useHttpClient}}httpParams = {{/useHttpClient}}httpParams.append(key,
```

Comment:
Might also be a date time

Reply:
Sorry, I did not understand. Handling for Date is the same as before. I'm not aware of a JS Datetime standard object...

Reply:
in the typescript-angular generator the `type: string; format: date-time` maps to the type `Date` in javascript, while `type: string; format: date` maps to `string` by default.
So here something like `{{^isDateTime}}.substr(0, 10){{isDateTime}}` should help

Reply:
I see now and changed it. Thank you!